.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -176,10 +176,12 @@ stages:
       displayName: Generate Matrices
       name: generate_variables_step
 
-- stage: build_windows_tracer
+- stage: build_and_package_windows
   dependsOn: [master_commit_id]
   variables:
     masterCommitId: $[ stageDependencies.master_commit_id.fetch.outputs['set_sha.master']]
+    # Temporarily use the profiling repo's commit hash for the beta MSI's filename
+    BetaMsiSuffix: $[ stageDependencies.build_windows_profiler.build.outputs['SetMsiSuffix.BetaMsiSuffix']]
```

Comment:
This doesn't make sense here - you've deleted the `build_windows_profiler` stage 🤔 I _think_ we can remove all the references to `BetaMsiSuffix` from everywhere now, as we don't ship a "beta" MSI, just the normal MSI

![image](https://user-images.githubusercontent.com/18755388/166437410-894f922f-2220-441f-a0c1-934d5928f8f5.png)

Reply:
Indeed. Fixed in 5edb6ffac1b5d96e5201336a81a5e56193263dd1